pkg/network/ovn_kubernetes.go

```diff
@@ -31,7 +31,7 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, manifestDir string) ([]*uns.U
 	data := render.MakeRenderData()
 	data.Data["ReleaseVersion"] = os.Getenv("RELEASE_VERSION")
 	data.Data["OvnImage"] = os.Getenv("OVN_IMAGE")
-	data.Data["HypershiftImage"] = os.Getenv("HYPERSHIFT_IMAGE")
+	data.Data["SDNControllerImage"] = os.Getenv("SDN_CONTROLLER_IMAGE")
```

Comment:
Oops, this was a cut+paste leftover. It can just go away; ovn-kubernetes doesn't use that image.

Reply:
done